Card/index.d.ts

```diff
@@ -1,43 +1,28 @@
+import Button from '../Button';
+import Icon from '../Icon';
 import MaterialComponent from '../MaterialComponent';
 import { VNode } from 'preact';
-import Button from '../Button';
-
-declare class Section<PropsType = {}, StateType = {}> extends MaterialComponent<PropsType & JSX.HTMLAttributes, StateType> {}
 
 declare interface ICardProps extends JSX.HTMLAttributes {
-  'theme-dark'?: boolean;
+  'stroked'?: boolean;
 }
 export default class Card extends MaterialComponent<ICardProps, {}> {
-  static Primary: typeof Primary;
-  static SupportingText: typeof SupportingText;
   static Actions: typeof Actions;
-  static Action: typeof Action;
+  static ActionButton: typeof ActionButton;
+  static ActionIcon: typeof ActionIcon;
   static Media: typeof Media;
-  static Title: typeof Title;
-  static Subtitle: typeof Subtitle;
-  static HorizontalBlock: typeof HorizontalBlock;
-  static MediaItem: typeof MediaItem;
```

Comment:
You forgot to add it to Card as `static CardMediaContent: typeof CardMediaContent`.